se/Mortgage/v0/schema/definitions/Address.yaml

```diff
@@ -0,0 +1,35 @@
+"$id": "https://open-broker.org/se/Mortgage/v0/schema/definitions/Address"
+"$schema": "http://json-schema.org/draft-07/schema#"
+
+title: Address
+description: An address for a physcical person
```

Comment:
Physical should describe "address", not "person" (although this looks more awesome)

Reply:
I meant person, like a physical person in contrast to a _legal person_. But maybe I can stick with just "An address" if it is confusing.

All though I misspelled _physical_ :stuck_out_tongue: